TABTestKit/Classes/Contexts/BiometricsContext.swift

```diff
@@ -6,6 +6,9 @@
 //
 
 import Foundation
+#if !COCOAPODS
+import Biometrics
```

Comment:
Can you explain a bit more about why this is needed?

Reply:
Before to answer of why this is needed. Let’s try to understand how Carthage and Cocoapods works.

Cocoapods allows developers to install only a part of the library by using Subspec, in our case. It’s `BDD` and `Biometrics`. But in Carthage world, it’s a different scenario. Carthage doesn’t support **Subspecs** unfortunately. It will build a framework based from the Xcode Project.

To have something similar on Carthage, I decided to create two extra frameworks with `BDD` and `Biometrics` that allows developers if they want to install only BDD or/and Biometrics. On our client project, we might want to have to use BDD.framework for Unit Tests outside the UI Test. TABTestKit will, of course, embed also BDD and Biometrics of course. 

Now, to answer to the question of why this is needed this conditional import, it’s because, in Carthage world, we will need to import BDD and Biometrics because TABTestKit doesn't include whereas, in Cocoapods world, TABTestKit won’t need it to import them because it will be embedded into one single framework TABTestKit.

The other alternative for us to not have this conditional import, it's having BDD & Biometrics including TABTestKit but it means it won't give the flexibility in Carthage world to have only BDD for Unit Testing for example.

P.S. : I added a screenshot where we can see the different scheme for TABTestKit, BDD and Biometrics. Also, selected the TABTestKit where we can see which frameworks TABTestKit needs to have some dependencies.

<img width="963" alt="Screenshot 2019-12-06 at 09 39 02" src="https://user-images.githubusercontent.com/3829527/70312816-4684bd80-180c-11ea-96c2-4ced2bd72da7.png">

Reply:
So could we simplify this for not having different components for Carthage and have subspecs as a Cocoapods-only feature?

Reply:
Made some changes. Now only one framework for Carthage library and for a Cocoapods library, we still have the option to use as subspaces.